sdk/resources/Azure.ResourceManager.Resources.Bicep/api/Azure.ResourceManager.Resources.Bicep.net8.0.cs

```diff
@@ -0,0 +1,66 @@
+        protected override bool TryGetTypeBuilderCore(System.Type type, out System.ClientModel.Primitives.ModelReaderWriterTypeBuilder builder) { throw null; }
+    }
+    public static partial class ResourcesBicepExtensions
+    {
+        public static Azure.Response<Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationSuccessResponse> BicepDecompileOperationGroup(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationSuccessResponse>> BicepDecompileOperationGroupAsync(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+}
+namespace Azure.ResourceManager.Resources.Bicep.Mocking
+{
+    public partial class MockableResourcesBicepSubscriptionResource : Azure.ResourceManager.ArmResource
+    {
+        protected MockableResourcesBicepSubscriptionResource() { }
+        public virtual Azure.Response<Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationSuccessResponse> BicepDecompileOperationGroup(Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationSuccessResponse>> BicepDecompileOperationGroupAsync(Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+}
+namespace Azure.ResourceManager.Resources.Bicep.Models
+{
+    public static partial class ArmResourcesBicepModelFactory
+    {
+        public static Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationSuccessResponse DecompileOperationSuccessResponse(System.Collections.Generic.IEnumerable<Azure.ResourceManager.Resources.Bicep.Models.FileDefinition> files = null, string entryPoint = null) { throw null; }
+        public static Azure.ResourceManager.Resources.Bicep.Models.FileDefinition FileDefinition(string path = null, string contents = null) { throw null; }
+    }
+    public partial class DecompileOperationContent : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationContent>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Resources.Bicep.Models.DecompileOperationContent>
```

Comment:
I think it would be more reasonable if we rename this to `DecompileBicepContent`.